src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -680,6 +682,11 @@ def setUp(self):
         if self.setup_start_agents:
             self.start_agents(force=force_agent_start)
 
+        # Write an ID string to the log file for cross rerferencing logs with
+        # tests
+        id_str = '"Test.name: ' + str(self) + '"'
+        self.write_string_to_logfile(id_str)
```

Comment:
This should be a conditional call.   Some tests will end up setting `self.setup_start_servers` to False and therefore there will not be a server running to communicate with.  

If it could be turned into a `DaosServerManager` method or input that may help it be more portable to tests that startup the daos_server at different times (see also https://github.com/daos-stack/daos/blob/master/src/tests/ftest/pool/destroy_tests.py - which starts multiple server groups post `setUp()`)

Reply:
d6953b615a5288d8518b8c215099e1b5e7c30482